web/app/scripts/recent-counts/recent-counts-directive.js

```diff
@@ -13,6 +13,6 @@
     }
 
     angular.module('driver.recentCounts')
-    .directive('recentCounts', RecentCounts);
+    .directive('driverRecentCounts', RecentCounts);
```

Comment:
We have a `driver` prefix on all the directives, so I just changed it to be consistent.